src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -9,89 +9,158 @@ namespace System.Collections.Immutable
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T>.Builder CreateBuilder<T>(int initialCapacity) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> CreateRange<T>(System.Collections.Generic.IEnumerable<T> items) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<TResult> CreateRange<TSource, TResult>(System.Collections.Immutable.ImmutableArray<TSource> items, System.Func<TSource, TResult> selector) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<TResult> CreateRange<TSource, TResult>(System.Collections.Immutable.ImmutableArray<TSource> items, int start, int length, System.Func<TSource, TResult> selector) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<TResult> CreateRange<TSource, TArg, TResult>(System.Collections.Immutable.ImmutableArray<TSource> items, System.Func<TSource, TArg, TResult> selector, TArg arg) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<TResult> CreateRange<TSource, TArg, TResult>(System.Collections.Immutable.ImmutableArray<TSource> items, int start, int length, System.Func<TSource, TArg, TResult> selector, TArg arg) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>() { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>(System.Collections.Immutable.ImmutableArray<T> items, int start, int length) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>(T item) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>(T item1, T item2) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>(T item1, T item2, T item3) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
         public static System.Collections.Immutable.ImmutableArray<T> Create<T>(T item1, T item2, T item3, T item4) { throw null; }
-        public static System.Collections.Immutable.ImmutableArray<T> Create<T>(params T[] items) { throw null; }
+        [System.Diagnostics.Contracts.PureAttribute]
+        public static System.Collections.Immutable.ImmutableArray<T> Create<T>(params T[]? items) { throw null; }
```

Comment:
This appeared to be a diff between the source and ref.  (ApiCompat doesn't yet catch nullable differences).

Reply:
/cc @buyaa-n

Reply:
Not sure how that missed, thanks for fixing it!